featurebyte/service/session_validator.py

```diff
@@ -115,7 +115,7 @@ async def _get_session(
             )
         session_manager = SessionManager(credentials={feature_store_name: credential})
         return await session_manager.get_session_with_params(
-            feature_store_name, session_type, details
+            feature_store_name, session_type, details, timeout=INTERACTIVE_SESSION_TIMEOUT_SECONDS
```

Comment:
If i understood this correctly, this method is called when calling `materialize_features`, `scheduled_materialize_features`, `initialize_new_columns`, `drop_columns` and `drop_table`. Is it expected to have shorted session timeout sections for all these cases?

Reply:
Those are use `_get_session` in `FeatureMaterializeService`.
This class `SessionValidatorService` is only used when creating a new feature store